roles/filetree_create/templates/current_inventory_sources.j2

```diff
@@ -53,6 +77,6 @@ controller_inventory_sources:
 {% endfor %}
 {% endif %}
 {% endfor %}
-{% if (last_inventory | default(true) | bool) and (last_inventory_source | default(true) | bool) %}
+{% if (last_inventory | default(true) | bool) or (last_inventory_source | default(true) | bool) %}
```

Comment:
Did you tested that change? It is not related with the PR itself, so this is why i'm asking...

Reply:
Change done by mistake, reverted.